deploy/osd-token-refresher/config.yaml

```
@@ -0,0 +1,4 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet:
+  matchExpressions:
+    - { key: api.openshift.com/fedramp, operator: DoesNotExist }
```

nit: can we split this out onto multiple lines so it's consistent with other config files

also, functionally, we should be checking for the true value rather than exists, because `api.openshift.com/fedramp: "false"` would actually this to not be pushed out either, which is incorrect/unintuitive

```suggestion
    - key: api.openshift.com/fedramp
      operator: NotIn
      value: 'true'
```